resources/views/dashboard/master.blade.php

```diff
@@ -56,6 +38,12 @@
             $('select').material_select();
           });
         </script>
+
+        <script src = "{!! load_asset('/js/jquery.min.js') !!}"></script>
+        <!-- Materialize Compiled and minified JavaScript -->
+        <script src = "{!! load_asset('/js/materialize.min.js') !!}"></script>
+        <script src = "{!! load_asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
+        <script src = "{!! load_asset('/js/scripts.js') !!}"></script>
         <script src = "{!! load_asset('/js/dashboard.js') !!}"></script>
```

Comment:
My reasoning was that `dashboard.js` is only needed in the admin dashboard so no need to merge and minify it. 

@unicodeveloper Let me do that just now